deploy/operator/hypershift/playbooks/spoke-crs-playbook.yaml

```diff
@@ -0,0 +1,57 @@
+- name: Create CRDs for Assisted Installer
```

Comment:
```suggestion
- name: Create CRs in order to deploy a spoke cluster via Assisted Installer
```

Reply:
same goes for other places where we mention CRDs instead of CRs (I know it's not your fault for having those mistakes in other files. Still let's have it correct at least for the new files :) )